src/pe_reports/report_generator.py

```diff
@@ -28,39 +21,221 @@
 
 # Third-Party Libraries
 import docopt
-import pkg_resources
-from pptx import Presentation
+import fitz
+import pandas as pd
 from schema import And, Schema, SchemaError, Use
+from xhtml2pdf import pisa
 
 from ._version import __version__
-from .pages import Pages
+from .data.db_query import connect, get_orgs
+from .pages import init
 
-# Configuration
-REPORT_SHELL = pkg_resources.resource_filename("pe_reports", "data/shell/pe_shell.pptx")
-REPORT_OUT = "Customer_ID_Posture_Exposure.pptx"
 
+def embed(
+    output_directory,
+    org_code,
+    datestring,
+    file,
+    cred_xlsx,
+    da_xlsx,
+    vuln_xlsx,
+    mi_xlsx,
+):
+    """Embeds raw data into pdf and encrypts file."""
```

Comment:
Another capitalization nitpick:
```suggestion
    """Embeds raw data into PDF and encrypts file."""
```

Reply:
Addressed [here](https://github.com/cisagov/pe-reports/pull/114/commits/9c39af6967a25d26434abdd25b958c107b77ccbd).